src/server/status/metrics_collector/cgroup.js

```diff
@@ -111,10 +111,10 @@ export function getAllStats(options = {}) {
       }
 
       return Promise.all([
-        readCPUAcctUsage(cpuAcctPath),
-        readCPUFsPeriod(cpuPath),
-        readCPUFsQuota(cpuPath),
-        readCPUStat(cpuPath)
+        readCPUAcctUsage(cpuAcctPath).catch(rejectUnlessFileNotFound),
+        readCPUFsPeriod(cpuPath).catch(rejectUnlessFileNotFound),
+        readCPUFsQuota(cpuPath).catch(rejectUnlessFileNotFound),
+        readCPUStat(cpuPath).catch(rejectUnlessFileNotFound)
```

Comment:
@tylersmalley I could use advice here - can we chat about this?

Without these changes, I was getting unhandled rejection warnings. They weren't causing the tests to fail, but they were pretty messy. Not sure what I've changed that makes the warnings quiet down after putting the catches up here.

Reply:
I just went ahead and undid this because I don't want to hack the code to get it to not complain about my mock module.

I still would really like to understand why the unhandled rejections are happening. This code is exactly like https://runkit.com/miu/5ad8d36c36503a0012574310